inc/RestApi/PluginsController.php

```diff
@@ -0,0 +1,353 @@
+<?php
+
+namespace Bluehost\Maestro\RestApi;
+
+use Exception;
+use WP_REST_Server;
+use WP_REST_Response;
+use Plugin_Upgrader;
```

Comment:
```suggestion
use Exception;
use WP_REST_Server;
use WP_REST_Response;
use Plugin_Upgrader;
```
We can remove these imports and wherever these are used just add a `\` to read them from the global namespace